Dockerfile

```diff
@@ -10,22 +10,19 @@
 #        /data/outprefix
 #
 
-FROM ubuntu:trusty
+FROM python:3.7-buster
 
 MAINTAINER Michael Perry <lmperry@stanford.edu>
 
 # Install dependencies
 RUN apt-get update && apt-get -y install \
-    python \
-    python-dev \
-    python-pip \
     jq \
     wget
 
 # Install scitran.data dependencies
 RUN pip install \
-  numpy==1.15.1 \
-  pydicom==0.9.9 \
+  "numpy>1.15.0,<1.16.0" \
+  "pydicom>2.0.0,<3.0.0" \
```

Comment:
I'll be in favor to pinning it to a specific version given how pydicom may change stuff on patch, minor version update (so that if we have a bug report, we don't have pydicom version discrepancies when debugging locally)

Reply:
Done